app/views/posts/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<div>
+  <%= @post.text %>
+</div>
```

Comment:
- [ ] Same here, Please add a placeholder written in plain HTML explaining what this view suppose to show.